Tests/SourceKitLSPTests/LocalSwiftTests.swift

```diff
@@ -618,12 +618,12 @@ final class LocalSwiftTests: XCTestCase {
     }, { (note: Notification<PublishDiagnosticsNotification>) in
       log("Received diagnostics for open - semantic")
       XCTAssertEqual(note.params.diagnostics.count, 1)
-      diagnostic = note.params.diagnostics.first!
+      diagnostic = note.params.diagnostics.first
     })
 
-    let request = CodeActionRequest(
+    let request = try CodeActionRequest(
       range: Position(line: 1, utf16index: 0)..<Position(line: 1, utf16index: 11),
-      context: CodeActionContext(diagnostics: [diagnostic], only: nil),
+      context: CodeActionContext(diagnostics: [XCTUnwrap(diagnostic, "expected diagnostic to be available")], only: nil),
```

Comment:
Nitpick, I would move the `try` in front of `XCTUnwrap` to make it clear that that’s the only function that can fail.
```diff
-   let request = try CodeActionRequest(
+   let request = CodeActionRequest(
       range: Position(line: 1, utf16index: 0)..<Position(line: 1, utf16index: 11),
       context: CodeActionContext(diagnostics: [diagnostic], only: nil),
-       context: CodeActionContext(diagnostics: [XCTUnwrap(diagnostic, "expected diagnostic to be available")], only: nil),
+       context: CodeActionContext(diagnostics: [try XCTUnwrap(diagnostic, "expected diagnostic to be available")], only: nil),
```

Similarly in a few more places